include/apple2.h

```diff
@@ -83,7 +83,57 @@
 #define CH_CURS_LEFT    0x08
 #define CH_CURS_RIGHT   0x15
 
-#if !defined(__APPLE2ENH__)
+#if defined(__APPLE2ENH__)
+
+/* MouseText-based functions for boxes and lines drawing */
+void mt_chline(unsigned char length);
+void mt_cvline(unsigned char length);
+void mt_chlinexy(unsigned char x, unsigned char y, unsigned char length);
+void mt_cvlinexy(unsigned char x, unsigned char y, unsigned char length);
+
+#define CH_HLINE        0x5F
+#define CH_VLINE        0xDF
+#define CH_ULCORNER     0x5F
+#define CH_URCORNER     0x20
+#define CH_LLCORNER     0xD4
+#define CH_LRCORNER     0xDF
+#define CH_TTEE         0x5F
+#define CH_BTEE         0xD4
+#define CH_LTEE         0xD4
+#define CH_RTEE         0xDF
+#define CH_CROSS        0xD4
+
+#define _chline(length)       mt_chline(length)
+#define _chlinexy(x,y,length) mt_chlinexy(x,y,length)
+#define _cvline(length)       mt_cvline(length)
+#define _cvlinexy(x,y,length) mt_cvlinexy(x,y,length)
+
+#else
+
+/* Functions that don't depend on MouseText to draw boxes and lines */
+void dyn_chline(unsigned char h, unsigned char length);
+void dyn_cvline(unsigned char v, unsigned char length);
+void dyn_chlinexy(unsigned char h, unsigned char x, unsigned char y, unsigned char length);
+void dyn_cvlinexy(unsigned char v, unsigned char x, unsigned char y, unsigned char length);
```

Comment:
Some nit picking: The (unusual) cc65 style for function prototypes is to have a space before the parameter list.